### ISSUE TITLE
Can not claim rewards anymore

### ISSUE THREAD

**Issue by ZILLI0N** [Zilliqa/Zillion-issue#38]
I used to be able to claim rewards but can't anymore: There is an error. Please try again.
![image](https://user-images.githubusercontent.com/71654421/94550274-c1afd800-0253-11eb-9fa5-99daa6105565.png)

**Comment by kensilverman** [comment]
Yes, I have confirmed on Atomic wallet sine Dec 15th the same issue, the issue appears to be the same, when pressing claim button,  claim does not go through posting an error message.

**Comment by kensilverman** [comment]
Further,  the staking balance is not increasing since about Dec 15th as well.

**Comment by Alex210386** [comment]
> Antes podía reclamar recompensas, pero ya no puedo: hay un error. Inténtalo de nuevo. ![imagen](https://user-images.githubusercontent.com/71654421/94550274-c1afd800-0253-11eb-9fa5-99daa6105565.png)

**Comment by Hubertus1711** [comment]
Yes same result after trying first Zillet.io -noresult-and now https://stake.zilliqa.com/ : " There is an error, please try again " also when i tried with enough gaslimit  (advised100000 but upgraded to 150000) enough zilliqa on wallet ( 350 + zilliqa) and gasprice( 200000000) Somebody knows a tip ? It is crazy because you can not get your zilliqa unstaked before you" Claim the rewards" which seems impossible. 
[ 2023-04-26 om .pdf](https://github.com/Zilliqa/Zillion-issue/files/11335573/2023-04-26.om.pdf)

**Comment by Thecityismine** [comment]
Do we assume the rewards and the staked Zill are lost forever?
